funnel/models/proposal.py

```diff
@@ -194,7 +198,7 @@ def status_title(self):
 
     @hybrid_property
     def confirmed(self):
-        return self.status == PROPOSALSTATUS.CONFIRMED
+        return self.status.value == PROPOSALSTATUS.CONFIRMED
```

Comment:
Remove this method entirely. Change all references to it from `proposal.confirmed` to `proposal.state.CONFIRMED`.